### DISCUSSION TITLE
Add post build checks that there is no /opt/ros

### DISCUSSION THREAD

**Discussion by asimonov** [space-ros/docker#278] [Ideas]
### Feature description

After every image build check that /opt/ros does not exist. If there is, it indicates that we missed an apt install of some ros package that in turn brings other dependencies. We should only build from source to not mix in packages from apt

### Implementation considerations

Post build workflow check

**Comment by Bckempa** [comment]
We should also inspect the workspace precedence of the colon generated `install.sh` scripts to make sure we don't have any unexpected underlays. If you look in the release images now you can see it actually sources the `/opt/ros/humble/` local install... oops.
